### PR TITLE
Updated API docs

### PR DESCRIPTION
**Pull request by du1ana**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,6 +24,7 @@
     "xrpl-accountlib": "3.2.9"
   },
   "devDependencies": {
-    "eslint": "8.3.0"
+    "eslint": "8.3.0",
+    "jsdoc-to-markdown": "^9.0.1"
```
Comment (kithminisg) [inline package.json:28]: Remove the caret and make it 9.0.1

File: src/clients/base-evernode-client.js [hunk 2]
```diff
@@ -91,7 +105,7 @@ class BaseEvernodeClient {
 
     /**
      * Connects the client to xrpl server and do the config loading and subscriptions. 'subscribe' is called inside this.
-     * @returns boolean value, 'true' if success.
+     * @returns Returns `true` if the connection is successful.
```
Comment (kithminisg) [inline src/clients/base-evernode-client.js:108]: Remove one `returns`

File: src/clients/base-evernode-client.js [hunk 3]
```diff
@@ -30,6 +30,20 @@ const REPUTATION_VALUE_PARAM_OFFSET = 20;
 // By reputation address <host_address(20)><token_id(32)><error(1)>
 const HOST_DEREG_FROM_REP_PARAM_SIZE = 53;
 
+/**
+ * Creates an instance of BaseEvernodeClient.
+ *
```
Comment (chalith) [inline src/clients/base-evernode-client.js:35]: Remove empty line

File: src/clients/foundation-client.js [hunk 4]
```diff
@@ -15,8 +15,21 @@ const REPUTATION_PARAM_SIZE = 21;
 
 const FoundationEvents = {}
 
+/**
+ * FoundationClient class to manage and interact with foundation operations.
+ * It extends the BaseEvernodeClient.
+ *
```
Comment (chalith) [inline src/clients/foundation-client.js:21]: Here also remove empty lines

**Review by chalith — APPROVED** [review]

**Review by chalith — APPROVED** [review]

**Review by kithminisg — APPROVED** [review]

**Pull request merged**